tests/test_layout.py

```diff
@@ -735,3 +738,69 @@ def test_tree_style_utf8(hlwm):
       … ├─╼ vertical: [FOCUS]
       … ╰─╼ vertical:
     """)
+
+
+def test_split_invalid_argument(hlwm):
+    wrongDecimal = [
+        '0.0f', '.3', '.', 'b',
+    ]
+    for d in wrongDecimal:
+        hlwm.call_xfail(['split', 'top', d]) \
+            .expect_stderr('invalid argument')
```

Comment:
I vaguely recall that we once missed a bug because we did not check if an expected error occurred for the right reason. Maybe we should verify the exception string here as well?

Reply:
> I vaguely recall that we once missed a bug because we did not check if an expected error occurred for the right reason. Maybe we should verify the exception string here as well?

First I was too lazy because the messages were different, but now I have extended it (and I added some more cases to test all exceptions possibly thrown by the `parse()`)